metaci/db/management/commands/populate_db.py

```diff
@@ -1,59 +1,71 @@
-from django.core.management.base import BaseCommand
+import random
 
+import factory.random
+from django.contrib.auth.models import Group
+from django.core.management.base import BaseCommand
 from django.db import transaction
-
 from guardian.shortcuts import assign_perm
 
-import factory.random
-
 from metaci import conftest as fact
-from metaci.users.models import User
-from metaci.testresults.models import TestResult
-from django.contrib.auth.models import Group
-
 from metaci.build.models import BuildFlow
-
-import random
+from metaci.cumulusci.models import Org
+from metaci.plan.models import Plan, PlanRepository
+from metaci.repository.models import Repository
+from metaci.testresults.models import TestResult
+from metaci.users.models import User
 
 
 class Command(BaseCommand):
     help = "Populates a testing DB with sample data"
 
-    def handle(self, *args, **options):
-        if self.check_non_destructive():
-            print("Populating test data")
-            # reset_db.Command().handle(*args, **options)
-            # migrate.Command().handle(*args, **options)
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--results", action="store_true", help="Create fake builds and test results"
+        )
 
-            self.create_objs()
-        else:
-            print(
+    def handle(self, *args, **options):
+        if options["results"]:
+            if self.check_non_destructive():
+                print("Populating test data")
```

Comment:
I'm not familiar with this code -- are print statements how MetaCI does logging? Or were these some debugging statements that were accidentally left in?

Reply:
All these specific lines you commented on are code that Paul wrote a while back and I didn't touch aside from moving inside a conditional.

Using a logger would be best but this code is in a command line command so the print function isn't doing any harm.

Reply:
fair enough.

Reply:
Why did you prefer this part of the data generation be optional?